src/lib/pages/backoffice/Document/DocumentEditor/DocumentEditor.js

```diff
@@ -17,10 +17,9 @@ export class DocumentEditor extends Component {
   constructor(props) {
     super(props);
 
-    const isEditing = !!props.match.params.documentPid;
     this.state = {
       data: {},
-      isLoading: isEditing,
+      isLoading: this.userOnEditPage(),
```

Comment:
Was the rename needed here?

Reply:
I like being verbose but that maybe just my style hehe

Reply:
I agree with verbosity! But here the name is referring to the document state and not the user action. Also, I think you don't need a function to evaluate that condition unless I am missing something!

Reply:
fair enough! then maybe  **userIsEditing** is better ? 

I made it into a function because the same logic was being repeated a lot (checking if there was a documentRequestPid in the url) and that did not seem that clean to me.

Reply:
Ok, now I see what you meant. Then, just extract that to a `get` method i.e
```javascript
get isUserEditing = () => !!this.props.match.params.documentPid;
```
then you can use it as `if(this.isUserEditing) {...}`

Reply:
good one!